tck/old-tck-selenium/ajax/src/main/webapp/tagwrapper/ajaxTagWrap.xhtml

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html
+        PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
+        "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<!--
+
+    Copyright (c) 2013, 2022 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Should copyrights be updated since these application files didn't change much (or very little, i.e. added id)